### DISCUSSION TITLE
Custom footer/copyright

### DISCUSSION THREAD

**Discussion by frederic-loui** [squidfunk/mkdocs-material#5685] [Q&A] [answered]
Hi ! I'm creating a document web site and i'm customizing the copyright right now. I'd tried to tweak and follow some discussions (find by googling the issue) and docs but could not find a "elegant" answer to the problem.  Do you know how to properly set image on the right ? For now this is what I have by using `overrides/partials/copyright.html`:

![Capture d’écran 2023-07-06 à 16 17 19](https://github.com/squidfunk/mkdocs-material/assets/9639687/ed786997-bcba-445d-8b6b-51791ebf338a)

**Comment by squidfunk** [comment, marked as answer]
Without knowing your exact HTML and CSS, it's hard to give exact directions, bug something like this should work

``` html
<div style="display: flex; justify-content: space-between; align-items: center">
  <div>Aligned to left</div>
  <div>Aligned to right</div>
</div>
```

> **Warning**: This example uses inline styles for clarity, but it's better to move this into CSS classes.

**Reply by frederic-loui** [reply]
Thanks for your answer and again sorry for the question. You might have noticed that I'm not a web developer.
Please find the code in `docs/overrides/partials/copyright.html`

```
<div class="md-copyright">
<div style="display: flex; justify-content: space-between; align-items: center">
  <div>Aligned to left</div>
  <div>Aligned to right</div>
</div>
    {% if config.copyright %}
    <div class="md-copyright__highlight">
        {{ config.copyright }}
    </div>
    {% endif %}
    {% if not config.extra.generator == false %}
    Made with
    <span class="twemoji heart">
        {% include ".icons/octicons/heart-fill-24.svg" %}
    </span> by
    <a href="https://github.com/orgs/rare-freertr/people" target="_blank" rel="noopener">
        RARE contributors</a> and <a href="https://squidfunk.github.io/mkdocs-material/">Material for MkDocs</a>
    {% endif %}
</div>
```
I just copy-paste the code you provided me in order to identify the right boundaries.
Resulting image is here below:

![Capture d’écran 2023-07-07 à 13 20 34](https://github.com/squidfunk/mkdocs-material/assets/9639687/4481cb93-1019-4a3b-baf1-30af09a5536f)

Basically I wanted to be able to put the text or the image on the right end side of the rectangle(right end of the page footer)

Maybe I should override `partials/footer.html` then ? 

Anyhow, thanks for this awesome piece of code. I can also accomodate to have everything on the LEFT as a work around.

Best/Frederic

**Comment by frederic-loui** [comment]
Hourra ! I overidden `footer.html` and that did the trick !
Thanks for your super help  @squidfunk.
 
![Capture d’écran 2023-07-07 à 14 14 28](https://github.com/squidfunk/mkdocs-material/assets/9639687/6da8ffbf-26f7-4056-baf0-22a876af457a)

Again, many thanks  again for this awesome piece of work !

**Reply by squidfunk** [reply]
Glad I could help!
